src/test/java/org/broadinstitute/consent/http/MockServerTestHelper.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.consent.http;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.platform.launcher.TestExecutionListener;
+import org.junit.platform.launcher.TestPlan;
+import org.mockserver.client.MockServerClient;
+import org.testcontainers.containers.MockServerContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+public class MockServerTestHelper extends AbstractTestHelper implements TestExecutionListener,
+    WithMockServer {
+
+  public static final MockServerContainer container = new MockServerContainer(IMAGE).waitingFor(
```

Comment:
Normally static final objects are uppercase
```suggestion
  public static final MockServerContainer CONTAINER = new MockServerContainer(IMAGE).waitingFor(
```